README.md

```diff
@@ -14,9 +14,69 @@
 NNI (Neural Network Intelligence) is a toolkit to help users run automated machine learning (AutoML) experiments. 
 The tool dispatches and runs trial jobs generated by tuning algorithms to search the best neural architecture and/or hyper-parameters in different environments like local machine, remote servers and cloud.
 
-<p align="center">
-<img src="./docs/img/nni_arch_overview.png" alt="drawing"/>
-</p>
+### **NNI [v0.5](https://github.com/Microsoft/nni/releases) has been released!**
+
+<table>
+  <tbody>
+    <tr>
+      <td colspan=2 align="center"><b>User Code + <a href="docs/howto_1_WriteTrial.md">SDK (import nni)</a></b></td>
+      <td colspan=2 align="center"><b>Tuning Algorithms</b></td>
+      <td colspan=2 align="center"><b>Training Services</b></td>
+    </tr>
+    <tr>
+      <td colspan=2>
+      <ul>
+        <li>CNTK</li>
+        <li>Tensorflow</li>
+        <li>PyTorch</li>
+        <li>Keras</li>
+        <li>...</li>
+        </ul>
+        (Python based frameworks)
+      </td>
+      <td colspan=2>
+      <a href="docs/HowToChooseTuner.md">Tuner</a>
+      <ul>
+        <li><a href="docs/HowToChooseTuner.md#TPE">TPE</a></li>
+        <li><a href="docs/HowToChooseTuner.md#Random">Random</a></li>
+        <li><a href="docs/HowToChooseTuner.md#Anneal">Anneal</a></li>
+        <li><a href="docs/HowToChooseTuner.md#Evolution">Naive Evolution</a></li>
+        <li><a href="docs/HowToChooseTuner.md#SMAC">SMAC</a></li>
+        <li><a href="docs/HowToChooseTuner.md#Batch">Batch</a></li>
+        <li><a href="docs/HowToChooseTuner.md#Grid">Grid</a></li>
+        <li><a href="docs/HowToChooseTuner.md#Hyperband">Hyperband</a></li>
+        <li><a href="docs/HowToChooseTuner.md#NetworkMorphism">Network Morphism</a></li>
+        <li><a href="examples/tuners/enas_nni/README.md">ENAS*</a></li>
```

Comment:
could we remove * from ENAS? and add Metis tuner since you mention v0.5?

Reply:
sure, updated.